src/Evolution/Systems/Cce/ScriPlusValues.cpp

```diff
@@ -0,0 +1,374 @@
+    const Scalar<SpinWeighted<ComplexDataVector, 1>>& eth_beta,
+    const Scalar<SpinWeighted<ComplexDataVector, 0>>& eth_ethbar_beta,
+    const Scalar<SpinWeighted<ComplexDataVector, -1>>& ethbar_eth_ethbar_beta,
+    const Scalar<SpinWeighted<ComplexDataVector, 2>>& dy_du_bondi_j,
+    const Scalar<SpinWeighted<ComplexDataVector, 1>>& ethbar_dy_du_bondi_j,
+    const Scalar<SpinWeighted<ComplexDataVector, 0>>& boundary_r,
+    const Scalar<SpinWeighted<ComplexDataVector, 1>>& eth_r_divided_by_r,
+    const size_t l_max, const size_t number_of_radial_points) noexcept {
+  const size_t number_of_angular_points =
+      Spectral::Swsh::number_of_swsh_collocation_points(l_max);
+
+  const SpinWeighted<ComplexDataVector, 0> exp_2_beta_at_scri;
+  make_const_view(make_not_null(&exp_2_beta_at_scri), get(exp_2_beta),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 1> eth_beta_at_scri;
+  make_const_view(make_not_null(&eth_beta_at_scri), get(eth_beta),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 0> eth_ethbar_beta_at_scri;
+  make_const_view(make_not_null(&eth_ethbar_beta_at_scri), get(eth_ethbar_beta),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, -1> ethbar_eth_ethbar_beta_at_scri;
+  make_const_view(make_not_null(&ethbar_eth_ethbar_beta_at_scri),
+                  get(ethbar_eth_ethbar_beta),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 1> eth_r_divided_by_r_view;
+  make_const_view(make_not_null(&eth_r_divided_by_r_view),
+                  get(eth_r_divided_by_r),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 2> dy_du_j_at_scri;
+  make_const_view(make_not_null(&dy_du_j_at_scri), get(dy_du_bondi_j),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  const SpinWeighted<ComplexDataVector, 1> ethbar_dy_du_j_at_scri;
+  make_const_view(make_not_null(&ethbar_dy_du_j_at_scri),
+                  get(ethbar_dy_du_bondi_j),
+                  (number_of_radial_points - 1) * number_of_angular_points,
+                  number_of_angular_points);
+
+  // Attempting the consensus form; math still needs re-examining
+  // extra factor of * -sqrt(2) to agree with SpEC tetrad normalization
+  get(*psi_3) = 2.0 * conj(eth_beta_at_scri) +
+                4.0 * conj(eth_beta_at_scri) * eth_ethbar_beta_at_scri +
+                ethbar_eth_ethbar_beta_at_scri +
```

Comment:
the doc has ethbar_ethbar_eth_beta

Reply:
I'll update the doc, but they are equivalent for beta (because it is spin 0)